src/lib/data/createables.ts

```diff
@@ -726,6 +726,26 @@ const Reverteables: Createable[] = [
 		},
 		noCl: true
 	},
+	{
+		name: 'Revert ancient icon',
+		inputItems: {
+			[itemID('Ancient icon')]: 1
+		},
+		outputItems: {
+			[itemID('Ancient essence')]: 5000
+		},
```

Comment:
You dont have to change it, but for future reference, these should be just banks. `new Bank().add()`